lib/index.js

```diff
@@ -178,6 +178,34 @@ function _ensure_cheerio(value) {
     return value;
 }
 
+class DictionaryAsset extends BaseAsset {
+    constructor() {
+        super();
+        this._object_type = "DictionaryObject";
+        this._content_type = "text/html";
```

Comment:
I would not reuse the blog template for this - just make some simple HTML document with no styling for the time being